pkg/controller/atlasdatabaseuser/databaseuser_test.go

```diff
@@ -52,46 +52,46 @@ func TestCheckUserExpired(t *testing.T) {
 	fakeClient := fake.NewClientBuilder().WithScheme(scheme).Build()
 
 	t.Run("Validate DeleteAfterDate", func(t *testing.T) {
-		result := checkUserExpired(zap.S(), fakeClient, "", *mdbv1.DefaultDBUser("ns", "theuser", "").WithDeleteAfterDate("foo"))
+		result := checkUserExpired(context.TODO(), zap.S(), fakeClient, "", *mdbv1.DefaultDBUser("ns", "theuser", "").WithDeleteAfterDate("foo"))
```

Comment:
is this really a TODO or can we wire `context.Background()` here? as per godoc Background is ok for tests specificially:

```
$ go doc context.Background
package context // import "context"

func Background() Context
    Background returns a non-nil, empty Context. It is never canceled, has no
    values, and has no deadline. It is typically used by the main function,
    initialization, and tests, and as the top-level Context for incoming
    requests.
```

Reply:
Ok, I will change to Background.

Reply:
Done